defaults/main.yml

```diff
@@ -12,11 +12,11 @@ datadog_user: dd-agent
 datadog_group: root
 
 # default apt repo
-datadog_apt_repo: "deb http://apt.datadoghq.com/ stable main"
+datadog_apt_repo: "deb http://apt.datadoghq.com/ stable 6"
```

Comment:
while we're at it (making breaking changes), can we switch the default apt repo to `https`? :)

We already install `apt-transport-https` to support https repos so it should work out-of-the-box